docs/developer/app-store/apps/adyen/pci-dss.mdx

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_label: PCI DSS
+sidebar_position: 6
+title: Adyen App - PCI DSS compliance
+---
+
+The following diagram shows the flow of sensitive data between services when using the Adyen App:
+
+```mermaid
+sequenceDiagram
+    box Merchant controlled
+    actor User via storefront
+    end
+    participant Saleor
+    participant Adyen App
+    participant Adyen API
+
+    rect rgba(240,150,150,0.5)
+        note right of User via storefront: Primary Account Number (PAN) Flow
+        User via storefront <<->> Adyen API: Pass sensitive data, including credit card number and PIIs<br/>to render the payment UI
+    end
+
+    User via storefront ->> Saleor: Pass encrypted sensitive payment data
+    create participant Saleor database
+    Saleor ->> Saleor database: Store internal transaction model (non sensitive)
+    Saleor ->> Adyen App: Pass through encrypted sensitive payment data
+    Adyen App ->> Adyen API: Pass through encrypted sensitive payment data
+    create participant Adyen App database
+    Adyen App ->> Adyen App database: Store internal transaction model (non sensitive)
+
+    Adyen API ->> Adyen App: Provide non-ecrypted payment data and encrypted payment data
+    Adyen App ->> Saleor: Pass non-ecrypted payment data and encrypted payment data  Adyen
```

Comment:
A word is missing, perhaps you meant the following?

```suggestion
    Adyen App ->> Saleor: Pass non-ecrypted payment data and encrypted payment data from Adyen
```

Reply:
Fixed in https://github.com/saleor/saleor-docs/pull/1608/commits/6bbb987eb838e6ff96262a65a8700f55ece5e2d0